contracts/test/mocks/MockCollateralizedTermsContract.sol

```diff
@@ -0,0 +1,207 @@
+/*
+
+  Copyright 2017 Dharma Labs Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity 0.4.18;
+
+import "./MockContract.sol";
+import "../../Collateralizer.sol";
+
+
+contract MockCollateralizedTermsContract is MockContract {
+    using SafeMath for uint;
+
+    // Maps from agreementId's to the amount repaid to date
+    mapping (bytes32 => uint) internal amountRepaid;
+    // Maps from agreementId's to a mapping of timestamps to the expected
+    // value repaid at said timestamp
+    mapping (bytes32 => mapping (uint => uint)) internal expectedValueRepaidAtTimestamp;
+    // Maps from agreementId's to a list of timestamps at which
+    // repayments are expected (used primarily for resetting our mocks)
+    mapping (bytes32 => uint[5]) internal expectedRepaymentSchedule;
+    // Maps from agreementId's to the number of repayment dates
+    // expected for that agreement
+    mapping (bytes32 => uint) internal numRepaymentTimestamps;
+    // Maps from agreementId's to the mocked term end timestamp
+    mapping (bytes32 => uint) internal termEndTimestamp;
+
+    function registerRepayment(
+        bytes32 agreementId,
+        address payer,
+        address beneficiary,
+        uint unitsOfRepayment,
+        address tokenAddress
+    )
+    public
```

Comment:
Nit: spacing should be consistent with other contracts in this repo (i.e. `public` and `returns ...` should be tab'd right)